wg-manifests/charter.md

```diff
@@ -1,70 +1,32 @@
-# WG Manifests Charter
+# WG Platform/Manifests Charter
```

Comment:
I'm not sure if we need to rename this working group.

There could be a separate need for a new working group that maintains platform stuff, but I think keeping the focus on the manifests is more sustainable for this working group.

Reply:
Right now or at least for the last 5 years our focus is the platform while the manifests make up less than 5 % of the work. Just copying manifests can be done by a robot. This is not what this WG is about.

Reply:
I agree with @thesuperzapper, the goal of Working Group is to own the code, and it is not clear what code will be maintained by this WG.

Reply:
This seems like a positive reason to discuss the creation of a specific working group for the "platform" components?

That way each group can have clear ownership of specific goals.

This also lets us have a community discussion around the scope of "Kubeflow Platform" (dashboard, auth, profiles, etc), rather than just putting an unwritten expectation on the Manifest and Notebooks WG maintainers (mostly you and me right now).

Reply:
I am totally willing to continue maintaining the status quo, whether it is written or not. I have changed the PR now so much, according to the comments, that it is probably worth reading it again and opening a new comment.